docker/test/integration/features/core_functionality.feature

```diff
@@ -67,3 +67,9 @@ Feature: Core flow functionalities
     When all instances start up
     Then the peak memory usage of the agent is more than 130 MB in less than 20 seconds
     And the memory usage of the agent decreases to 70% peak usage in less than 20 seconds
+
+  Scenario: Metrics can be logged
+    Given a GenerateFlowFile processor
+    And log metrics publisher is enabled in MiNiFi
+    When all instances start up
+    Then the Minifi logs contain the following message: '{"LogMetrics":{"RepositoryMetrics":{"flowfile":{"running":"true","full":"false","size":"0"},"provenance":{"running":"true","full":"false","size":"0"}}}}' in less than 30 seconds
```

Comment:
Can we format this log message in a more human-friendly way?

Reply:
Updated to use `PrettyWriter` for the log output in 33895d1fe45f18c6a82ab0efe76ee480161cc4db

Reply:
I meant logging something like an English sentence instead of dumping JSON.

Reply:
I'm not sure that's a good idea. In this simple case it looks readable, but in case there are multiple metrics with a lot of subnodes, like AgentInformation that has a large hierarchy of nodes, or QueueMetrics that can have a large number of connections depending on the number of processors, the output can become unreadable, and hard to find the metric you are looking for. I think the hierarchical structure of the JSON can help with this and can be easily read.